crates/ruff_python_parser/src/parser/statement.rs

```diff
@@ -413,6 +413,17 @@ impl<'src> Parser<'src> {
 
         let exc = if self.at(TokenKind::Newline) {
             None
+        } else if self.at(TokenKind::From) {
```

Comment:
nit: with multiple `self.at` calls, I'd normally move it to a `match` instead:

```rs
let exc = match self.current_token_kind() {
	TokenKind::Newline => {}
	TokenKind::From => {}
	_ => {}
};
```

Reply:
I went with the `match`, but next bikeshed color is yours, Micha, I promise!